src/App.js

```diff
@@ -197,10 +197,9 @@ function App(props) {
           <Route path={"/"} exact={true}>
             <NearOrgPage
               {...passProps}
-              src={Widgets.nearOrg.homePage}
+              src={signedIn ? Widgets.activityPage : Widgets.nearOrg.homePage}
```

Comment:
wouldnt this essentiall make NearOrgPage our new main page component instead of ViewPage? I dont think that's ideal, can we do a conditional rendering of ViewPage instead?

Reply:
Great point! Will adjust